modular_skyrat/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -7,11 +7,14 @@
 
 //Service
 /datum/job/assistant
-	alt_titles = list("Civilian", "Visitor", "Businessman", "Trader", "Entertainer")
+	alt_titles = list("Civilian", "Visitor", "Businessman", "Trader", "Entertainer", "Morale Officer", "Public Mining Worker", "Trainee", "D.A.B. Operative")
```

Comment:
Plz nuh
```suggestion
	alt_titles = list("Civilian", "Visitor", "Businessman", "Trader", "Entertainer", "Trainee")
```

Reply:
No idea why you asking change about PMW given the fact assistants can easily access public mining. Morale Officer is "big brain name" for Entertainer, and D.A.B. Operative exist because DAB Suit and Helmet exist, why not direct that complaint to their existence?
But whatever, if you don't want these - you won't want entire PR.